test/framework/cleanup.go

```diff
@@ -0,0 +1,60 @@
+// Copied from https://github.com/kubernetes/kubernetes/blob/de66c643c479f420b1902f8897636ea41ac2b3c3/test/e2e/framework/cleanup.go
```

Comment:
Is it necessary to copy that file - cant we use a package import?

Reply:
Let's please not start vendoring k/k - it's a pain. cc @mvladev